code/mob/living/critter/gunbot.dm

```diff
@@ -93,3 +94,46 @@
 
 	get_disorient_protection()
 		return max(..(), 80)
+
+	attack_hand(mob/user)
+		user.lastattacked = src
+		if(!user.stat)
+			if (user.a_intent != INTENT_HELP)
+				actions.interrupt(src, INTERRUPT_ATTACKED)
+			switch(user.a_intent)
+				if(INTENT_HELP) //Friend person
+					playsound(src.loc, 'sound/impact_sounds/Generic_Shove_1.ogg', 50, 1, -2)
+					user.visible_message("<span class='notice'>[user] gives [src] a [pick_string("descriptors.txt", "borg_pat")] pat on the [pick("back", "head", "shoulder")].</span>")
+				if(INTENT_DISARM) //Shove
+					SPAWN_DBG(0) playsound(src.loc, 'sound/impact_sounds/Generic_Swing_1.ogg', 40, 1)
```

Comment:
Why spawn?

Reply:
Good question!
I copied it from robots
https://github.com/goonstation/goonstation/blob/02c202ec1f52df4dc72e723dc8f989a8828e4997/code/mob/living/silicon/robot.dm#L1523

Reply:
I'd yeet the spawn then.
```suggestion
					playsound(src.loc, 'sound/impact_sounds/Generic_Swing_1.ogg', 40, 1)
```